src/roman_datamodels/stnode.py

```diff
@@ -404,7 +423,7 @@ def select_tag(self, obj, tags, ctx):
         return obj.tag
 
     def to_yaml_tree(self, obj, tag, ctx):
-        return obj._data
+        return dict(obj._data)
```

Comment:
```suggestion
        return obj._data
```
I think this also can be reverted.